design/ingressroute-design.md

```diff
@@ -387,9 +387,31 @@ The best way I've thought to mitigate this so far is the vertex ingress routes w
 
 This is unfortunate for two reasons:
 
-- It's more boilerplate: a deletates to b, b has to list a as a cross check.
+- It's more boilerplate: a delegates to b, b has to list a as a cross check.
 - It turns a DAG into a linked list (possibly not the right term), but it would be impossible for a web service to be a member of multiple roots, unless of course we made the list of incoming vertices be a list -- which would probably push the solution into using virtualhost.fqdn.
 
 The last thing is that this boilerplate _should be required_ even when not in enforcing mode.
 I'm not interested in proposing a design where the security interlock that prevents dave's cheap webhosting from publishing gmail without TLS is considered to be the customer's problem because they did not add an optional key. 
 Said again, if this is the mitigation we choose to adopt, it has to be mandatory for all users, because we all know how effective optional security features are.
+
+# Metrics
```

Comment:
Does it make sense to have equivalent metrics for the regular `Ingress` resource?

Reply:
All ingress routes should be provided already by kube-state-metrics, also Envoy has built-in metrics for clusters which would map to ingress resources.

Reply:
Aaah right. And we can't use kube-state-metrics for IngressRoute because it doesn't support CRDs. Thanks!